src/setup-guild.ts

```diff
@@ -33,4 +38,15 @@ async function deployCommands(guild: Guild): Promise<void> {
 	await rest.put(Routes.applicationGuildCommands(guild.members.me!.id, guild.id), {
 		body: [...commands, ...contextMenus],
 	});
+}
+
+async function setupSlowMode(guild: Guild): Promise<void> {
+	const slowModes = await SlowMode.findAll({
+		where: {
+			enabled: true
+		},
+		include: { all: true }
+	});
+
+	slowModes.forEach(slowMode => handleSlowMode(guild, slowMode));
 }
```

Comment:
I feel that this could go in `ready.ts` instead because it's not inherently guild-specific. `handleSlowMode` could take a channel instead of a guild as an argument if `client.channels.fetch()` is used.